backend/src/main/java/atlantbh/restaurants/controllers/MenuItemController.java

```diff
@@ -0,0 +1,15 @@
+package atlantbh.restaurants.controllers;
+
+import atlantbh.restaurants.models.MenuItem;
+import atlantbh.restaurants.services.MenuItemService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class MenuItemController extends BaseController<MenuItem, MenuItemService> {
+    public ResponseEntity itemsByMenu(@PathVariable Long menu) {
```

Comment:
filter.. queryParam menuId={id}

Reply:
Fixed in 0cd4debab433260ab6a8267aa0b72b31db581edb